pkg/securitypolicy/open_door.rego

```diff
@@ -1,14 +1,14 @@
 package policy
 
-api_svn := "0.10.0"
+api_svn := "@@API_SVN@@"
```

Comment:
Should we have a framework_svn here?

Reply:
Not needed, as the open door policy doesn't use the framework.